data/Remnant Cognizance.txt

```diff
@@ -0,0 +1,626 @@
+	on enter Nenia
+		dialog `You have never seen such activity from the Void Sprites. Many quickly congregate around the strange ship, and seem to escort it to Nasqueron. It seems like that is where it wanted to be. In the distance, you can see the archon methodically pulverizing a korath ship, presumably another raider.`
+	on stopover
+		conversation
+			`You follow the creature down through the clouds of Nasqueron and emerge into what can only be described as an amphitheater in the clouds. Thousands of void sprites fill the area and the creature you have been following floats near the center of it. Your external sensors pick up myriad pulses of energy and eerie whistling noises that seem to resonate in the clouds. As the <ship> moves forward there is a sudden silence and the flickers of energy synchronize.`
+			`	You feel a sudden pressure in your mind, and a sense of presence as the world around you fades out somewhat.`
+			`	"Know you not, do I. Live creature inside dead creature, like ones who flew but not everywhere." You are momentarily disoriented as you look around and see hundreds of views of yourself peering out the cockpit window. "Your intention reverberates - Know and not know - Return with more echoes - perhaps I shall hear."`
+			`	With that, the pressure vanishes and you are left staring out the window at the rapidly dispersing flock of void sprites. You aren't entirely sure what the voice meant, but you are quite sure that the Remnant are going to want to hear about this.`
+	on complete
+		event "remnant: nenia elder"
+
+
+
+event "remnant: nenia elder"
+	system "Nenia"
+		add fleet "Void Sprites Elder" 10000
+	
+
+
+mission "Remnant: Cognizance 14"
+	name "Go collect some void sprites"
+	description "Escort the pelican fleet to Nenia and back to Arculus"
+	source
+		government "Remnant"
+	landing
+	destination Viminal
+	waypoint "Nenia"
+	to offer
+		has "Remnant: Cognizance 13: done"
+	on offer
+		conversation
+			`It is almost starting to feel commonplace to have remnant pouncing on your ship and using your cockpit as a meeting room. This time, it is Prefect Chilia and Plume who seem to appear out of nowhere requesting permision to board. The two gather around your console to watch the recording of your trip. They nod at your narration of what transpired, and focus intently when you describe the strange presence in your mind. When you finish recounting your experience, they lean back in thought.`
+			`	"Captain <first>, the exact meaning is unclear, but it seems to be the voice of something not acustomed to speaking to us or creatures like us. Regardless, I think we should move forward with our plan" signs Prefect Chilia. "Best case scenario, we can deprive the Korath of easy access to the void sprites. Worst case scenario, we can move enough that - should the Korath hunt them to the extinction in Nenia, they will continue to exist in our territory."`
```

Comment:
`	"Captain <first>, the exact meaning is unclear, but it seems to be the voice of something not accustomed to speaking to us or creatures like us. Regardless, I think we should move forward with our plan," signs Prefect Chilia. "Best case scenario, we can deprive the Korath of easy access to the void sprites. Worst case scenario, we can move enough that - should the Korath hunt them to the extinction in Nenia, they will continue to exist in our territory."`

Reply:
Fixed.